force-app/src/classes/MethodSpy.cls

```diff
@@ -152,37 +162,55 @@ global class MethodSpy {
   }
 
   global interface MethodSpyCall {
```

Comment:
suggestion: rename to `MethodSpyConfig`

Reply:
I think it is going to be hard as it is global and exposed and customers can use this type...

Reply:
Let keep the name stable then